bgpy/tests/framework_tests/unit_tests/test_scenario.py

```diff
@@ -237,6 +238,99 @@ def test_get_announcements(self, engine):
         for attacker_asn in scenario.attacker_asns:
             assert any(attacker_asn == x.seed_asn for x in scenario.announcements)
 
+    def test_no_attackers(self, engine):
+        """
+        Ensures that get_attacker_asns is empty if override_attacker_asns is an empty
+        set.
+
+        This trivial test ensures that the method does not regress to previous
+        behavior, where a random attacker was selected.
+        """
+        # Sample announcement sent from just the victim
+        asn = ASNs.VICTIM.value
+        anns = (Announcement(prefix="1.2.0.0/16", as_path=tuple([asn]), seed_asn=asn),)
+        override_victims = frozenset[int]({asn})
+        override_attackers = frozenset[int]()
+
+        config = ScenarioConfig(
+            ScenarioCls=SubprefixHijack,
+            num_attackers=0,
+            override_victim_asns=override_victims,
```

Comment:
Is there a reason you're overriding victim asns here when it's not tested? Same question with overriding announcements, both for this test and the next test

Reply:
Oh that's a good point, I don't need to override both. I overrode the announcements because SubprefixHijack errors when there isn't one victim